rest-api/offline/sa_key_remove.py

```diff
@@ -9,6 +9,7 @@
 
 def delete_service_account_keys():
   days_to_delete = config.getSetting(config.DAYS_TO_DELETE_KEYS)
+  service_account_keys = config.getSetting(config.SERVICE_ACCOUNT_KEYS)
```

Comment:
I believe this should be:

service_account_keys = config.getSettingList(config.SERVICE_ACCOUNT_KEYS, default=[])

getSettingList() allows us to return lists instead of individual values; default ensures this doesn't fail if the config setting is missing. (Which it would be for everything but test and staging.)